docs/advanced_usage.rst

```diff
@@ -241,6 +241,62 @@ next step of the pipeline or ``None`` if the trace should be discarded::
 
 (see filters.py for other example implementations)
 
+.. _`Logs Injection`:
+
+Logs Injection
+--------------
+
+Datadog APM traces can be integrated with Logs by first having the tracing
+library patch the standard library ``logging`` module and updating the log
+formatter used by an application. This feature enables the user to reference a
+trace from a log entry emitted within a trace context.
+
+Before the trace information can be injected into logs, the formatter has to be
+updated to include ``dd.trace_id`` and ``dd.span_id`` attributes from the log record.
+The integration with Logs occurs automatically as long as the format of logs is
+appended with ``- dd.trace_id=%(dd.trace_id)s dd.span_id=%(dd.span_id)s``.
```

Comment:
is `- ` in the spec? required?

Reply:
No it is not -- I've simplified the doc to make clear all that is necessary is what seems to work from the spec.